heaps/heap_sort.py

```diff
@@ -1,8 +1,17 @@
+from heapq import heappush, heappop
 
 
 def heap_sort(list):
```

Comment:
✨ Nice! How might you implement this using the `MinHeap` you created below?

And what is the time and space complexity?